### PR TITLE
Update purple haze theme and fix some unstyled elements

### PR DESCRIPTION
**Pull request by grafixeyehero**
After
Screenshot_2019-05-24 Jellyfin(3)
Screenshot_2019-05-24 Jellyfin(1)
Screenshot_2019-05-24 Jellyfin
Screenshot_2019-05-24 Jellyfin(2)

### CODE REVIEW

**Comment by dkanada** [comment]
@DrPandemic I was going to overhaul the item details page after the scroller so that might change in the near future.

**Comment by grafixeyehero** [comment]
@DrPandemic what about For Primary `#0ce8d6` and secondery `#D8E6FF`

**Comment by grafixeyehero** [comment]
ImageDownloader template changes
Before
Screenshot (before)
After
Screenshot (after)

alternate colors
Screenshot_2019-05-25 Jellyfin
alternate colors hover
Screenshot hover

**Comment by DrPandemic** [comment]
I think alternate color 3 looks really nice. Same for the label placement

File: src/components/imagedownloader/imagedownloader.template.html [hunk 1]
```diff
@@ -10,15 +10,15 @@ <h3 class="formDialogHeaderTitle">
         <div class="flex align-items-center justify-content-center flex-wrap-wrap" style="margin: 2em 0;">
 
             <div style="margin: 0;">
-                <label for="selectImageProvider" style="display: inline-block;">${LabelSource}</label>
-                <select id="selectImageProvider" style="padding-left:.5em;padding-right:0;display: inline-block;width:auto!important;">
+                <div class="selectContainer">
```
Comment (DrPandemic) [inline src/components/imagedownloader/imagedownloader.template.html]: You forgot to close the `div`. This break the page for me.

**Comment by EraYaN** [comment]
Please make sure before merging that the PR title fully describes the changes in the PR and that there are no typos and it follows the correct format.

**Comment by grafixeyehero** [comment]
Upate Label, Primery and Secondery btn, Header h1,h2,h3 and Cheackbox

Screenshot_2019-05-29 Jellyfin(2)
Screenshot_2019-05-29 Jellyfin(1)
Screenshot_2019-05-29 Jellyfin
Screenshot_2019-05-29 Jellyfin(3)

**Comment by grafixeyehero** [comment]
@DrPandemic try to remove cache on ur browser

**Comment by grafixeyehero** [comment]
@DrPandemic Yeah you are right  .emby-checkbox:checked+span+.checkboxOutline; there r duplicate code i am updating know. And last thing is there a news about imagedownload page?

**Comment by DrPandemic** [comment]
You were right, it was the cache. It took me like 10 force reloads and private browsing for it to fix itself. So, that imagedownloader is good.

But for the checkbox I still see a border when a checkbox is not focused. I'm personally not a fan and it doesn't align with other themes, but if it's the style you are going for, I'll +1 this. Is it?

**Comment by anthonylavado** [comment]
Whenever this is good to go @jellyfin/web, feel free to merge

**Comment by grafixeyehero** [comment]
> 
> 
> Whenever this is good to go @jellyfin/web, feel free to merge

**Comment by dkanada** [comment]
Not a fan of the text colors but I think everything else is excellent. The white text looked really nice for this theme in my opinion.

**Pull request merged**